ImperatorToCk3/Source/Imperator/Families/Families.h

```diff
@@ -0,0 +1,25 @@
+#ifndef IMPERATOR_FAMILIES_H
+#define IMPERATOR_FAMILIES_H
+#include "newParser.h"
+
+namespace ImperatorWorld
+{
+	class Family;
+	class Families : commonItems::parser
+	{
+	public:
+		Families() = default;
+		Families(std::istream& theStream); // For testing
```

Comment:
you don't need it.
in testing open with:
Familes newFamilies;
newFamilies->loadFamilies(someInputStream);

We use those constructors only when necessary - when regular constructors are hardwired to open some specific file and we need to override them with a custom stream, and you're not doing so here.

Reply:
I think I got it, but please check commit f2ebdd2.

Reply:
yup, that's it.